src/index.html

```diff
@@ -11,8 +11,55 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute is linking to a '.scss' file. Ensure that this file is being preprocessed into a CSS file, as browsers cannot directly interpret SASS files. If this is intended to be a CSS file, make sure the correct file is linked.